docs/api/utilities.md

```diff
@@ -0,0 +1,412 @@
+---
+sidebar_position: 2
+---
+
+# Standalone Utilities
+
+Individual classes and utilities that can be used independently of the main `SaikiAgent` class.
```

Comment:
we can have a high level utilities section and add mcpmanager reference as a sub-doc

don't need the other utilities